public/style/vet.css

```
@@ -0,0 +1,610 @@
+  font-size: 1rem;
+}
+.vet-recovery-pill[data-ready="1"] {
+  border-color: var(--vet-gold-bright);
+  box-shadow: var(--vet-shadow-gold), var(--vet-shadow-card);
+  animation: vetPillPulse 1.6s ease-in-out infinite;
+}
+.vet-recovery-pill:hover {
+  transform: scale(1.05);
+  border-color: var(--vet-gold-bright);
+}
+@keyframes vetPillPulse {
```

Stylelint (17.10.0)

[error] Expected keyframe name "vetPillPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)